realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -68,6 +68,28 @@ protected TableView(Context context, Table parent, long nativePtr, TableQuery qu
         this.query = query;
     }
 
+    /**
+     * Creates a copy of already created Java TableView Object.
+     * The method is not supposed to be called by the user of the db. The method is for internal use only.
+     *
+     * @param tableView An existing TableView
+     * @param nativeViewPtr A pointer to a clone of the existing TableView.
+     */
+    protected TableView(TableView tableView, long nativeViewPtr) {
```

Comment:
Why `protected` and not `private` ?
I would rename the variables to be more telling: `tableView` -> `originalTableView`  and `nativeViewPtr` -> `newTableViewPtr` or something like that.